DOC-5654 - in reference to the Corda 4 API, changed 'backwards compat… (#1594)
corda/corda-docs-portal: content/en/archived-docs/corda-os/4.7/api-stability-guarantees.md

# API stability guarantees

> Corda makes certain commitments about what parts of the Corda 4 API will preserve stability as they change and

Reviewer: ```suggestion
Corda makes certain commitments about which parts of the Corda 4 API will preserve stability and
```
This is to match the 4.8 file.

> which will not. Over time, more of the Corda 4 API will fall under the stability guarantees. Thus, APIs can be categorized in the following 2 broad categories:

Reviewer: ```suggestion
which will not. Over time, more of the Corda 4 API will fall under the stability guarantees. Thus, APIs can be categorized in the following two broad categories:
```
